src/web/app/js/api.js

```diff
@@ -0,0 +1,77 @@
+$(() => {
+
+    let ws = new WebSocket("ws://"+window.location.hostname+":8080/");
+
+    //função para carregar as mensagens ao iniciar a pagina
+    window.onload = () => {
+        if (localStorage.getItem('key') === null) {
+            $.getJSON("js/data.json", (data) => {
+                for(let i = 0; i < data.length; i++) {
+                    messageLeft(data[i]);
+                }
+            });
+        } else {
+            let buffer = JSON.parse(localStorage.getItem('key'));
+
+            for(let i = 0; i < buffer.length; i++) {
+                messageLeft(buffer[i]);
+            }
+        }
+    };
+
+    ws.onopen = () => {
+        let greetings = {
+            command: "greetings",
+            sessionID: document.cookie.slice("SessionID=".length)
+        };
+        ws.send(JSON.stringify(greetings));
+    };
+
+    ws.onclose = function() {
+        alert("Closed!");
+    };
+
+    ws.onerror = function(err) {
+        alert("Error: " + err);
+    };
+
+    ws.onmessage = (evt) => {
+        let data = evt.data;
+
+        if ((data.indexOf('{') === -1) && (data !== "Hello Webbrowser")) {
```

Comment:
Mds, isso é muito gambiarra. Ao invés de resolver/retirar ou pedir pra retirar vc fez isso, incrível

```suggestion
        if ((data.indexOf('{') === -1)) {
```

Remove tbm o bloco try/catch inteiro q envia o Hello Webbrowser, aquilo deveri ter sido retirado a um tempo

Mas adiciona um check (se n tiver) no seu js, tipo um try/catch na hora de dar parse no json, pro programa não der crash qdo não vier um json